tests/suites/1_stateful/00_stage/00_0012_stage_priv.result

```diff
@@ -0,0 +1,14 @@
+==== check internal stage write priv ===
```

Comment:
Also can move this to `/0_stateless/18_rbac/`?

Reply:
ok